### PR TITLE
Add Story and Comment for Project Page

### PR DESCRIPTION
**Pull request by DarkFlamingo**

### CODE REVIEW

File: frontend/src/scenes/ProjectPage/components/Story.tsx [hunk 1]
```diff
@@ -1,9 +1,74 @@
...
 
-const Story: FC = () => (
-  <div>
-    Story
-  </div>
-);
+interface StoryProps {
+  story: string;
+  privileges: { amount: number; privilege: string }[];
+  tags: string[];
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/ProjectPage/components/Story.tsx]: I believe, privileges and tags shouldn't be a part of Story.

File: frontend/src/scenes/ProjectPage/components/Story.tsx [hunk 2]
```diff
@@ -1,9 +1,74 @@
...
+      <Col xs={12} lg={8}>
+        Lorem Ipsum is simply dummy text of the printing and typesetting
+        industry. Lorem Ipsum has been the industrys standard dummy text ever
+        since the 1500s, when an unknown printer took a galley of type and
+        scrambled it to make a type specimen book. It has survived not only five
+        centuries, but also the leap into electronic typesetting, remaining
+        essentially unchanged. It was popularised in the 1960s with the release
+        of Letraset sheets containing Lorem Ipsum passages, and more recently
+        with desktop publishing software like Aldus PageMaker including versions
+        of Lorem Ipsum.s
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/ProjectPage/components/Story.tsx]: We need to insert HTML here (you can use https://www.npmjs.com/package/react-render-html or analogs)

File: frontend/src/scenes/ProjectPage/components/Story.tsx [hunk 3]
```diff
@@ -1,9 +1,74 @@
...
+    'fsadfdas',
+    'fsaddfas',
+    'fsadfsad',
+    'fsadfasf',
+    'fsdafasd',
+    'fsdafsa',
+    'fdsafasdfas',
+    'fdsafsad',
+    'fsdafsad'
+  ];
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/ProjectPage/components/Story.tsx]: Do you need this data if you already pass the project.tags?

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/ProjectPage/components/Story.tsx (hunk 1 above)
Comment (DarkFlamingo) [reply frontend/src/scenes/ProjectPage/components/Story.tsx]: This is not a finished block, I sent a pull request so I can go to another branch.

**Pull request merged**